src/renderer/src/routes/(main)/apps-view/components/ListOfApps.svelte

```diff
@@ -16,7 +19,7 @@
 </script>
 
 <div
-	class="align-center flex grow justify-center bg-light-background bg-fixed dark:bg-apps-list-dark-gradient"
+	class="align-center bg-light-background dark:bg-apps-list-dark-gradient flex grow justify-center bg-fixed"
```

Comment:
please run yarn lintAndFormatAll to fix this, maybe we can add a git hook to run this before push

Reply:
It's interesting that this happened - it must be a result of prettier because I did not do that myself.

Reply:
yes, it is automatically doing it, I am looking how to fix this, it's this
https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/113